src/services/pallets/PalletsStakingProgressService.ts

```diff
@@ -88,7 +97,10 @@ export class PalletsStakingProgressService extends AbstractService {
 		if (electionLookAhead.eq(new BN(0))) {
 			// no offchain solutions accepted
 			toggle = null;
-		} else if ((eraElectionStatus as { isClose?: boolean }).isClose) {
+		} else if (
+			eraElectionStatus &&
+			(eraElectionStatus as { isClose?: boolean }).isClose
```

Comment:
```suggestion
			(eraElectionStatus as { isClose?: boolean })?.isClose
```
Does that work?

Reply:
We should be able to eliminate that `eraElectionStatus &&` check with the `?` operator